Sources/SwiftDocC/DocumentationService/Convert/Fallback Link Resolution/ConvertServiceFallbackResolver.swift

```diff
@@ -0,0 +1,44 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+/// A resolver that attempts to resolve local references that wasn't included in the catalog or symbol input.
+///
+/// The ``ConvertService`` builds documentation for a single page at a time. If this page's content contains references to other local symbols, pages, or
+/// assets that aren't included in the original ``ConvertRequest``, this fallback resolver resolves those references.
+///
+/// The ``ConvertService`` only renders the one page that it provided inputs for. Because of this, the content that this fallback resolver returns is considered
```

Comment:
```suggestion
/// The ``ConvertService`` only renders the one page that it is provided inputs for. Because of this, the content that this fallback resolver returns is considered
```

Reply:
I think adding "is" here changes the meaning of the sentence. It is the convert service that provides values so this sentence is meant to read as "the convert service only renders the one page that _the convert service_ provided inputs for"